docs/extras/integrations/llms/ollama.ipynb

```diff
@@ -0,0 +1,224 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Ollama\n",
+    "\n",
+    "[Ollama](https://ollama.ai/) allows you to run open-source large language models, such as Llama 2, locally.\n",
+    "\n",
+    "Ollama bundles model weights, configuration, and data into a single package, defined by a Modelfile. It optimizes setup and configuration details, including GPU usage.\n",
+    "\n",
+    "This example goes over how to use LangChain to interact with an Ollama instance. For a complete list of supported models and model variants, see the [Ollama model library](https://github.com/jmorganca/ollama#model-library).\n",
+    "\n",
+    "## Setup\n",
+    "\n",
+    "First, follow [these instructions](https://github.com/jmorganca/ollama) to set up and run a local Ollama instance.\n",
+    "\n",
+    "## Usage"
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
Minor nit:

I set `--verbose` when running `ollama run llama2` and see a nice readout of tok usage, but we loose this here.

Reply:
It's possible, generation info is in the final chunk.

Let's add as a follow-up/as requested since we'd have to change this to subclass the more complex `BaseLLM` to pass it through.